runbooks/upp-content-es-runbook.md

```diff
@@ -0,0 +1,96 @@
+# UPP - Content Elasticsearch Cluster
+
+Elasticsearch cluster for use by the Content Search API Port service and Content-RW-Elasticsearch service. 
+
+## Code
+
+upp-content-es-cluster
+
+## Service Tier
+
+Platinum
+
+## Lifecycle Stage
+
+Production
+
+## Delivered By
+
+content
+
+## Supported By
+
+content
+
+## Known About By
+
+- hristo.georgiev
+- robert.marinov
+- elina.kaneva
+- georgi.ivanov
+- tsvetan.dimitrov
+- dimitar.terziev
+
+## Host Platform
+
+AWS
+
+## Architecture
+
+There are two regional Amazon Elasticsearch clusters, one in eu-west-1 and one in us-east-1. Each regional cluster contains 4 data nodes and 3 dedicated master nodes.
+
+## Contains Personal Data
+
+No
+
+## Contains Sensitive Data
+
+No
+
+## Failover Architecture Type
+
+ActiveActive
+
+## Failover Process Type
+
+PartiallyAutomated
+
+## Failback Process Type
+
+PartiallyAutomated
+
+## Failover Details
+
+The service is part of both Delivery clusters. The failover guide for Delivery clusters can be found here: https://github.com/Financial-Times/upp-docs/tree/master/failover-guides/delivery-cluster
```

Comment:
Put the `<` `>` around all URLs:
<https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses>